src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[],
+  onSelected: (person: Person) => void,
+};
+
+export const DropdownMenu: React.FC<Props> = React.memo(
```

Comment:
You should use React.memo only when you really have some problems and need to reduce component re-renders by parent